src/DropdownReference/DropdownReference.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.dropdown.DropdownReference" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../../../xsd/widget.xsd">
+    <name>Dropdown reference</name>
+    <description/>
+    <icon></icon>
+    <properties>
+        <property key="entityPath" type="entity" isPath="yes" pathType="reference">
+            <caption>Entity path</caption>
+            <category>Data source</category>
+            <description/>
+        </property>
+        <property key="attribute" type="attribute" entityProperty="entityPath">
+            <caption>Attribute</caption>
+            <category>Data source</category>
+            <description>The attribute from which selectable values will be retrieved</description>
+            <attributeTypes>
+                <attributeType name="AutoNumber"/>
+                <attributeType name="Enum"/>
+                <attributeType name="Decimal"/>
+                <attributeType name="Integer"/>
+                <attributeType name="Long"/>
+                <attributeType name="String"/>
+                <attributeType name="Boolean"/>
+            </attributeTypes>
+        </property>
+        <property key="editable" type="enumeration" defaultValue="default">
+            <caption>Editable</caption>
+            <category>Data source</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="default">Default</enumerationValue>
+                <enumerationValue key="never">Never</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="emptyOptionCaption" type="translatableString" required="false">
```

Comment:
I think is better to provide an empty message by default

Reply:
Not setting any is equivalent to empty.

Reply:
@McDoyen I meant.. default value like "No data found" or something like that